### PR TITLE
Clean up 08-family-of-maps

### LINKED ISSUE #117: Update naming in 08-families-of-maps
Many terms in `08-families-of-maps.rzk.md` seem to be named using an outdated naming scheme.

For example the name `total-equiv-family-of-equiv` is written "conclusion-last" and is used for what I believe should be called `is-fiberwise-equiv-is-total-equiv` or `is-family-of-equiv-is-equiv-total-map` or some variation thereof.

Some names follow the "conclusion-first" paradigm, but mix up `equiv` and `is-equiv`.
For example `family-equiv-total-equiv` takes a family of maps and an `is-equiv (total-map)`, but returns not fiberwise `is-equiv`, but rather actual fiberwise equivalences `Equiv`. It should probably be called something like `family-equiv-is-equiv-total-map`.

I was a bit scared to just run off on my own and try to clean up the document, since a) it is long and b) probably many other things rely on it.

So what is the best policy for this? As a preliminary measure, one could add properly named synonyms for the most important "outward facing" terms so that at least the dirt is contained until one can do a proper project-wide cleaning.

### PR DESCRIPTION
**Pull request by TashiWalde**
This is a housekeeping PR for 08-family-of-maps:
- Update many names to our current naming scheme. This of course created many ripples in the whole codebase. This resolves #117.
- Some code formatting and clean up.
- Restructured and added some prose to the section on fundamental theorem of identity types.

While I feel mostly confident that I chose good names, it might still be a good idea if someone like @fredrik-bakke could give it a glance to make sure I didn't do anything silly.

### CODE REVIEW

File: src/hott/08-families-of-maps.rzk.md [hunk 1]
```diff
@@ -16,109 +16,98 @@ maps.
   ( A : U)
   ( B C : A → U)
   ( f : (a : A) → (B a) → (C a))
-  : ( Σ (x : A) , B x) → (Σ (x : A) , C x)
-  := \ z → (first z , f (first z) (second z))
+  : ( total-type A B) → (total-type A C)
```
[the remark this answers is not in the document]
Comment (fredrik-bakke) [reply src/hott/08-families-of-maps.rzk.md:19]: For very common definitions like this, I consider it okay to have a discrepancy between what denotes the actual type, and what lemmas refer to it as. Although, using `Σ` in lemma names is also fine by me. If it is indeed consistency in naming you are going for, then I think `type-total` would be more consistent with the general naming scheme.

File: src/hott/08-families-of-maps.rzk.md [hunk 2]
```diff
@@ -16,109 +16,98 @@ maps.
...
-#def total-map-from-fiber
+#def fib-fiberwise-fib-total-map
   ( A : U)
   ( B C : A → U)
   ( f : (a : A) → (B a) → (C a))
-  ( w : (Σ (x : A) , C x))
-  : fib (Σ (x : A) , B x) (Σ (x : A) , C x) (total-map A B C f) w →
-    fib (B (first w)) (C (first w)) (f (first w)) (second w)
+  : ( (a , c) : total-type A C)
+  → fib (total-type A B) (total-type A C) (total-map A B C f) (a , c)
```
Comment (fredrik-bakke) [inline src/hott/08-families-of-maps.rzk.md:36]: Same as above, `map-total` for the functorial action on maps of sigmas, or `map-Σ`. However, there's also the binary action to consider. In `agda-unimath` we call `map-total` just `tot`, but I think `map-total` is better. The binary action is called `map-Σ`

File: src/hott/08-families-of-maps.rzk.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (fredrik-bakke) [reply src/hott/08-families-of-maps.rzk.md:19]: Gotcha! Then maybe we can use the special notation `Σ'` for this? What do you think? :)

[the remark this answers is not in the document]
Comment (TashiWalde) [reply src/hott/08-families-of-maps.rzk.md:19]: I have no strong opinion on `total-type` vs `Σ'`. Do you think it is worth the effort to switch from the latter to the former throughout the project?

**Review by fredrik-bakke — APPROVED** [review]
The namings are very sensible, thank you so much for doing this!

File: src/hott/08-families-of-maps.rzk.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (fredrik-bakke) [reply src/hott/08-families-of-maps.rzk.md:19]: If you are using VSCode, you can match the appropriate places by turning on regular expressions in your text search and searching for `(?<!-)\btotal-type\b(?!-)`. Then you can do a simple textual search-and-replace :)

File: src/hott/08-families-of-maps.rzk.md (hunk 2 above)
Comment (fredrik-bakke) [reply src/hott/08-families-of-maps.rzk.md:36]: That's reasonable, I just figured this was an appropriate occasion to bring it up.

File: src/hott/08-families-of-maps.rzk.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (fredrik-bakke) [reply src/hott/08-families-of-maps.rzk.md:19]: Don't worry about it though, I can do it another time.

**Pull request merged**